cms/static/js/views/components/add_library_content.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Provides utilities to open and close the library content picker.
+ *
+ */
+define(['jquery', 'underscore', 'gettext', 'js/views/modals/base_modal'],
+function($, _, gettext, BaseModal) {
+    'use strict';
+
+    var AddLibraryContent = BaseModal.extend({
+        options: $.extend({}, BaseModal.prototype.options, {
+            modalName: 'add-component-from-library',
+            modalSize: 'lg',
+            view: 'studio_view',
+            viewSpecificClasses: 'modal-add-component-picker confirm',
+            // Translators: "title" is the name of the current component being edited.
+            titleFormat: gettext('Add library content'),
+            addPrimaryActionButton: false,
+        }),
+
+        initialize: function() {
+            BaseModal.prototype.initialize.call(this);
+            // Add event listen to close picker when the iframe tells us to
+            window.addEventListener("message", function (event) {
+                if (event.data?.type === 'pickerComponentSelected') {
+                    var requestData = {
+                        library_content_key: event.data.usageKey,
+                        category: event.data.category,
+                    }
+                    this.refreshFunction(requestData);
+                    this.hide();
+                }
+            }.bind(this), { once: true }, false);
+        },
+
+        /**
+         * Adds the action buttons to the modal.
+         */
+        addActionButtons: function() {
+            this.addActionButton('cancel', gettext('Cancel'));
```

Comment:
This can be fixed in a fast-follow PR if you want, but it's not a great UX that the "Next" button is sometimes hidden until you scroll, and the Next + Cancel buttons are in totally different places. I think they need to either be both inside the modal or both in the legy UI actions bar.
![sceenshot](https://github.com/user-attachments/assets/d1a02f68-2943-4b21-8fef-c4c83da77d83)

Reply:
Actually, do we even need a `Next` button? Selecting a course should directly take to next page since we only have one item to select in the first slide.

Reply:
Removed next button and updated it to go to component selection on library selection.


https://github.com/user-attachments/assets/776403dc-a865-4568-a74a-809786c50efa

Reply:
Makes sense to me!